src/docs/error-handling.dox

```diff
@@ -29,6 +29,18 @@ and it's unspecified when or even if exclusive access to the object will be
 granted. Generally, applications will not call WiredTiger methods requiring
 exclusive access when the objects might be in active use by other threads.
 
+@section error_sub_level Sub-level errors
+
+From the addition of WT_SESSION::get_last_error function and sub-level error codes, users
```

Comment:
How about
```suggestion
With the addition of the WT_SESSION::get_last_error function and sub-level error codes, users
```

Reply:
Thanks fixed it [5058d85](https://github.com/wiredtiger/wiredtiger/pull/11427/commits/5058d85dc0806548b7d763abfb402c0bc84f4be2)